presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -57,6 +60,9 @@
 
     private boolean debug;
 
+    private final Map<String, ProgressStatus> previousStageStatuses = new HashMap<>();
+    private ProgressStatus previousStatementStats = new ProgressStatus(start, 0, 0);
```

Comment:
Why called `previousStatementStats` instead of `previousStatementStatus`? (As compared with `previousStageStatuses`? ) :)

Alternatively, we can call everything `Stats`: `ProgressStats`, `previousStageStats` and `previousStatementStats`, just be consistent :)